docs/Solana/DataFeeds/using-data-feeds-solana.md

```diff
@@ -4,26 +4,41 @@ section: solana
+  "See the available data feeds on Solana":"/docs/solana/data-feeds-solana/"
+}
 metadata:
   title: "Using Data Feeds On-Chain (Solana)"
   description: "How to use Chainlink Data Feeds in your on-chain Solana programs."
 ---
 
-Chainlink Data Feeds are the quickest way to connect your smart contracts to the real-world market prices of assets. This guide demonstrates how to deploy a contract to the Solana Devnet and access Data Feeds on-chain using the [Chainlink Solana Starter Kit](https://github.com/smartcontractkit/solana-starter-kit). To learn how to read price feed data using off-chain applications, see the [Using Data Feeds Off-Chain](/docs/solana/using-data-feeds-off-chain/) guide.
+Chainlink Data Feeds are the quickest way to connect your smart contracts to the real-world market prices of assets. This guide demonstrates how to deploy a program to the Solana Devnet cluster and access Data Feeds on-chain using the [Chainlink Solana Starter Kit](https://github.com/smartcontractkit/solana-starter-kit). To learn how to read price feed data using off-chain applications, see the [Using Data Feeds Off-Chain](/docs/solana/using-data-feeds-off-chain/) guide.
 
-To get the full list of available Chainlink Data Feeds on Solana, see the [Solana Feeds](/docs/solana/data-feeds-solana/) page. View the program that owns the Chainlink Data Feeds in the [Solana Devnet Explorer](https://solscan.io/account/HEvSKofvBgfaexv23kMabbYqxasxU3mQ4ibBMEmJWHny?cluster=devnet).
+To get the full list of available Chainlink Data Feeds on Solana, see the [Solana Feeds](/docs/solana/data-feeds-solana/) page. View the program that owns the Chainlink Data Feeds in the [Solana Devnet Explorer](https://solscan.io/account/HEvSKofvBgfaexv23kMabbYqxasxU3mQ4ibBMEmJWHny?cluster=devnet), or the [Solana Mainnet Explorer](https://solscan.io/account/HEvSKofvBgfaexv23kMabbYqxasxU3mQ4ibBMEmJWHny).
 
 {% include 'data-quality.md' %}
 
-## Overview
+**Table of contents:**
+
+- [The Chainlink Data Feeds OCR2 Program](#the-chainlink-data-feeds-ocr2-program)
+- [Using the Solana Starter Kit](#using-the-solana-starter-kit)
+- [Adding Data Feeds On-Chain In An Existing Project](#adding-data-feeds-on-chain-in-an-existing-project)
+
+## The Chainlink Data Feeds OCR2 Program
+
+The program that owns the data feeds on both Devnet and Mainnet is [HEvSKofvBgfaexv23kMabbYqxasxU3mQ4ibBMEmJWHny](https://solscan.io/account/HEvSKofvBgfaexv23kMabbYqxasxU3mQ4ibBMEmJWHny?cluster=devnet). This is the program ID that you use to retrieve Chainlink Price Data on-chain in your program. The source code for this program is available in the [smartcontractkit/chainlink-solana](https://github.com/smartcontractkit/chainlink-solana/tree/develop/contracts/programs/ocr2) repository on GitHub.
+
+You can [add data feeds to an existing project](#adding-data-feeds-on-chain-in-an-existing-project) or [use the Solana Starter Kit](#using-the-solana-starter-kit).
+
+## Using the Solana Starter Kit
```

Comment:
as per the off-chain page, my comment questioning the order of starter kit vs add to existing applies here too